cli/src/wallet_cli/mod.rs

```diff
@@ -314,7 +314,25 @@ impl FromStr for OutputSelector {
 
 // `accounts` command
 pub async fn accounts_command(wallet: &Wallet) -> Result<(), Error> {
-    print_outputs(wallet.data().await.accounts().cloned().collect(), "Accounts:")
+    let wallet_data = wallet.data().await;
+    let accounts = wallet_data.accounts();
+
+    println_log_info!("Accounts:\n");
+
+    for account in accounts {
+        let output_id = account.output_id;
+        let account_id = account.output.as_account().account_id_non_null(&output_id);
+        let account_address = account_id.to_bech32(wallet.client().get_bech32_hrp().await?);
+
+        println_log_info!(
+            "{:<16} {output_id}\n{:<16} {account_id}\n{:<16} {account_address}\n",
+            "Output ID:",
+            "Account ID:",
+            "Account Address:"
+        );
+    }
```

Comment:
can this be factored out into a function bc you do the same for the implicit accounts, right?